BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -751,3 +787,11 @@ extension NSTextCheckingResult.CheckingType {
     }
 }
 
+extension AttributedLabel {
```

Comment:
This might very well be too clever by half but without adding explicit localization to the project this seems like the most reliable way to get this string. 
We could allow a localized string to be injected into the label but that feels like a slightly different code smell and even so I'd like to offer this as a default to prevent a breaking API change.  

I'm actively looking for thoughts and suggestions on this one.